docs/integrations/skopt.rst

```diff
@@ -51,101 +106,90 @@ Everything can be inspected live.
     results = skopt.forest_minimize(objective, space, callback=[neptune_callback],
                                     base_estimator='ET', n_calls=100, n_random_starts=10)
 
-Log all results
----------------
-You can log additional information from skopt results after the sweep has completed.
-By running:
+Step 5: Monitor your Skopt tuning in Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now you can switch to the Neptune tab which you had opened previously to watch the tuning live!
 
-.. code-block:: python3
+|Run on Colab|
 
-    skopt_utils.log_results(results)
+.. _skopt-advanced-options:
 
-You log the following things to Neptune:
+More Options
+------------
```

Comment:
I'd add/divide this in sections:

- Log diagnostic charts after the sweep -> Basically what you did but I'd first show the results of running this .log_results and than add a paragraph (or a note) about log_plots and log_pickle arguments